v2/mongodb-to-googlecloud/src/main/java/com/google/cloud/teleport/v2/mongodb/templates/MongoDbToBigQuery.java

```diff
@@ -128,6 +134,7 @@ public static boolean run(Options options)
             MongoDbIO.read()
                 .withUri(mongoDbUri)
                 .withDatabase(options.getDatabase())
+                .withQueryFn(FindQuery.create().withFilters(filter))
```

Comment:
let's do not add this config to config when the filter is null, empty, or {}. There might be performance implications.